JSON_for_IO-Link.yaml

```diff
@@ -3306,7 +3341,6 @@ components:
     masterCapabilitiesGet:
       required:
         - numberOfPorts
```

Comment:
It might be a problem to remove a previously required property. 
What do you think of adding a new property?
`numberOfWirelessPorts`

Reply:
Creating a new property instead? I cannot follow that argument!

Reply:
Yes, but what is the purpose of the additional property?

Reply:
Will be discussed in a new pull request.